src/site/components/error-message.md-njk

```diff
@@ -11,7 +11,8 @@ Show an error message next to the field and in the error summary when there is a
 
 {{ guidanceComponentPreview({
   componentName: "date-input",
-  exampleName: "with-whole-date-error"
+  exampleName: "with-whole-date-error",
+  macroOptions: macroOptions["error-message"]
```

Comment:
This one is a bit weird as the options relate to the actual error message component. It's how it is on https://design-system.service.gov.uk/components/error-message/. It's linked to from other components.

Reply:
I think it the correct way, the macros has some generic variable names that could clash